packages/dumb/src/components/trends/common/trends-container/trends-container.tsx

```diff
@@ -0,0 +1,109 @@
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { type FunctionComponent } from 'react'
+import { TrendsSvgContainerSized as TrendsSvgContainer } from '../trends-svg-container/trends-svg-container'
+import { type BgPrefs } from '../../../../models/blood-glucose.model'
+import type MedicalDataService from 'medical-domain'
+import { type Cbg, DatumType, Unit } from 'medical-domain'
+import { type RangeSegmentSlice } from '../../../../models/enums/range-segment.enum'
+import { type FocusedSlice } from '../../../../models/focused-slice.model'
+import { type OnSelectDateFunction } from '../../../../models/on-select-date-function.model'
+import { extent } from 'd3-array'
+import { connect } from 'react-redux'
+import _ from 'lodash'
+import { type ActiveDays } from '../../../../models/active-days.model'
+
+interface TrendsState {
+  focusedCbgSlice: FocusedSlice
+  focusedCbgSliceKeys: RangeSegmentSlice[]
+  showingCbgDateTraces: boolean
+}
+
+interface TrendsContainerProps {
+  currentCbgData: []
+  days: string[]
+  activeDays: ActiveDays
+  bgPrefs: BgPrefs
+  tidelineData: MedicalDataService
+  onSelectDate: OnSelectDateFunction
+  trendsState: TrendsState
+  // Only for redux
+  currentPatientInViewId: string
+}
+
+const MGDL_CLAMP_TOP = 400
+const MMOLL_CLAMP_TOP = 22.5
+
+const TrendsContainer: FunctionComponent<TrendsContainerProps> = (props) => {
+  const {
+    currentCbgData,
+    days,
+    activeDays,
+    bgPrefs,
+    tidelineData,
+    onSelectDate,
+    trendsState
+  } = props
+
+  const yScaleClampTop = {
+    [Unit.MilligramPerDeciliter]: MGDL_CLAMP_TOP,
+    [Unit.MmolPerLiter]: MMOLL_CLAMP_TOP
+  }
+  const upperBound = yScaleClampTop[bgPrefs.bgUnits]
+  // FIXME The `MedicalDataService.grouped` getter should be improved to allow stricter typing
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is there a JIRA ticket for this? There should be, and it should be referenced in this FIXME

Reply:
I haven't created it because I did not know if it was worth it or not... What do you think? I don't know how these kinds of technical improvements are handled

Reply:
IMO: if it is not worth having a JIRA ticket, then it is probably not worth doing it. And having it as a comment in the code is not useful.